interp/gta.go

```diff
@@ -233,7 +233,7 @@ func (interp *Interpreter) gta(root *node, rpath, importPath string) ([]*node, e
 						name = pkgName
 					}
 					name = filepath.Join(name, baseName)
-					if _, exists := sc.sym[name]; !exists {
+					if _, exists := sc.sym[name]; interp.allowRedecl || !exists {
```

Comment:
I think we can always tolerate a  re-import of an identical package in yaegi, as packages to parse are de-duplicated in any case. Adding an `AllowRedecl` option to interp is useless to me, there is no sense nor value to be too strict here, beyond your use case. I suggest to remove the definition of `AllowRedecl` option and instead verifying that if the symbol already exists, it matches the existing one:
```
if sym, exists := sc.sym[name]; exists && sim.kind == pkgSym && sym.typ.cat == srcPkgT && sym.typ.path == ipath {
    break
}

Reply:
👍